lib/rules/web/ocm_console/cluster_detail.xyaml

```diff
@@ -702,3 +702,21 @@ subscription_settings_error_message_loaded:
         selector:
             xpath: //div[contains(., '<error_message>')]
         timeout: 30
+wait_cluster_addon_uninstallation_on_detail_page:
+    element:
+        selector:
+            xpath: //div[@class='pf-l-grid pf-m-gutter']//span[text()='Add-on uninstallation']/../../dd/div[text()='<cluster_status>']
+        timeout: <wait_time>
+wait_cluster_uninstallation_on_detail_page:
+    element:
+        selector:
+            xpath: //div[@class='pf-l-grid pf-m-gutter']//span[text()='Cluster uninstallation']/../../dd/div[text()='<cluster_status>']
+        timeout: <wait_time>
+wait_for_uninstallation_to_finish:
```

Comment:
So here my logic is like this: after the uninstallation is finished, the UI page will be automatically redirected to the cluster list page, there is not successful messages or something like that. So I think if we want to wait for the uninstallation to finish, we can simply wait for the cluster list page to show.

Reply:
No. You need to check the cluster list page show and cluster you deprovisioned missing. And you do not need to wait at cluster detail page because you didn't need to check the page redirection. You can just goto cluster list page and wait for cluster missing.